src/Validation/Validator.php

```diff
@@ -260,13 +260,13 @@ public function validateParams(array $formDetails, bool $strictValidation = true
 						break;
 					// Check validation for min number value.
 					case 'min':
-						if ((string) $dataValue > (string) $inputValue) {
+						if ((string) $dataValue > (string) $inputValue && !empty($inputValue)) {
 							$output[$paramKey] = \sprintf($this->getValidationLabel('validationMin', $formId), $dataValue);
 						}
 						break;
 					// Check validation for min number value.
 					case 'max':
-						if ((string) $dataValue < (string) $inputValue) {
+						if ((string) $dataValue < (string) $inputValue && !empty($inputValue)) {
```

Comment:
Would it make more sense to bail out earlier if the `$inputValue` is empty?

Also, why are you comparing strings? I mean they will be numerically compared, but still I think you said that you're comparing numbers.

Reply:
This can be a minus number, float or empty so it was easier to compare strings. 

for empty check this will validate only if value is populated so we don't validate empty fields that are not required.